doc/manual/drtio.rst

```diff
@@ -20,90 +14,41 @@ The lower layers of DRTIO are similar to White Rabbit, with the following main d
 * no Ethernet compatibility
 * only star or tree topologies are supported
 
-From ARTIQ kernels, DRTIO channels are used in the same way as local RTIO channels.
-
-.. _using-drtio:
-
-Using DRTIO
------------
+Time transfer and clock syntonization is typically done over the serial link alone. The DRTIO code is written as much as possible to support porting to different types of transceivers (Xilinx MGTs, Altera MGTs, soft transceivers running off regular FPGA IOs, etc.) and different synchronization mechanisms.
 
 Terminology
-+++++++++++
+-----------
 
-In a system of interconnected DRTIO devices, each RTIO core (driving RTIO PHYs; for example a RTIO core would connect to a large bank of TTL signals) is assigned a number and is called a *destination*. One DRTIO device normally contains one RTIO core.
+In a system of interconnected DRTIO devices, each RTIO core is assigned a number and is called a *destination*. One DRTIO device normally contains one RTIO core.
```

Comment:
the term "PHY" isn't explained anywhere else in the manual and never otherwise referenced except for two of the argparse generated descriptions in core_reference -- IMO the parenthetical isn't really adding anything in this specific section, which is supposed to be about the DRTIO-specific terminology, and only confuses the sentence. if we want to concretely establish the correspondence RTIO core = PHYs somewhere for people who already know what a PHYs is and/or the general definition of RTIO core for those who don't, then this page doesn't seem like the place for it (could add a sentence to that effect in "RTIO Concepts", maybe.)

Reply:
Makes sense, will do